test/validators/no-returns-required/purpose.validator.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const PurposeValidator = require('../../../app/validators/return-requirements/purpose.validator.js')
+
+describe('Purpose validator', () => {
+  describe('when valid data is provided', () => {
+    const payload = {
+      purposes: [
+        'Heat Pump',
+        'Horticultural Watering',
+        'Hydraulic Rams',
+        'Hydraulic Testing',
+        'Hydroelectric Power Generation'
+      ]
+    }
+
+    it('confirms the data is valid', () => {
+      const result = PurposeValidator.go(payload)
+
+      expect(result.value).to.exist()
+      expect(result.error).not.to.exist()
+    })
+  })
+
+  describe('when invalid data is provided', () => {
```

Comment:
Could these tests be fleshed out a bit more to add more validation errors like the user not picking any options?